halconfig/settings.js

```diff
@@ -170,17 +170,32 @@ window.spinnakerSettings = {
   gitSources: ['bitbucket', 'gitlab', 'github', 'stash'],
   maxPipelineAgeDays: maxPipelineAgeDays,
   notifications: {
+    bearychat: {
+      enabled: true,
+    },
     email: {
       enabled: true,
     },
+    githubStatus: {
+      enabled: true,
+    },
+    googlechat: {
+      enabled: true,
+    },
     hipchat: {
       botName: 'Skynet T-800',
       enabled: true,
     },
+    pubsub: {
+      enabled: true,
+    },
+    slack: {
```

Comment:
It seems like we're removing configurability here, is that intentional?

The `slack` object that used to be assigned to this field includes template literals that I assume halyard populates?

If it's intentional then let's also remove the slack object defined on line 30.

Reply:
Good catch - this is a copy/paste error.

Reply:
Fixed.